src/styles/typo/_typo-webfonts.scss

```diff
@@ -5,7 +5,7 @@
 /// @param {string} $base-path [$font-base-path] - The path where your webfonts are stored.
 @mixin publico-headline($base-path: $font-base-path) {
   @font-face {
-    font-family: 'Publico Headline Web';
+    font-family: 'Publico Headline';
```

Comment:
@LucaMele 
@MKaHead 
May we should check with the font supplier?
Because the `PublicoHeadlineWeb.zip` file we received, explicitly states `font-family: 'Publico Headline Web';` in their "How to install" instructions.
@LucaMele I forwarded that email to you containing zip file about 10 months ago.
I also wrote them an email that they have invalid `font-style` values, so maybe there are more issue we don't know yet 😕

Reply:
btw. below it's also defined as for src by by both `local('Publico Headline Web'),` and `local('Publico Headline'),`

Reply:
@AndyOGo everywhere is used "Publico Headline" in the css (plib and aem) therefore this is not a breaking change. The other way around (naming it with Web) will instead create braking changes. We decied to do it this way, regardless of the name of the zip. Thank you

Reply:
yepp ok thank you @AndyOGo :)